src/MacroTools/Mechanics/DemonGates/DemonGateBuff.cs

```diff
@@ -85,10 +85,11 @@ public DemonGateBuff(unit caster, int demonUnitTypeId, float spawnInterval, int
     public override void OnApply()
     {
       Target
-        .IssueOrder("setrally", Target.GetPosition())
+        .IssueOrder(852002, Target.GetPosition())
```

Comment:
It's great that you've switched over to the ID version of this (ideally I'd like to delete the string version once all uses of it are gone), when we only have a raw ID to supply I think we want to maintain readability by using theconversion, so like (I don't remember the actual conversion function off the top of my head).

What would probably be even better is having an `OrderConstants` static class that contains these. It would, I think, only need to contain the weird ones like Set Rally, since `Constants.cs` already generates them for all abilities.

`.IssueOrder(ToOrderId("setrally"), Target.GetPosition())`

Reply:
Amended - added OrderId("setrally")